src/utils/isPRLabelValid.js

```diff
@@ -0,0 +1,19 @@
+// Ignore list of labels for frogtoberfest, add at end of array for more list
+const FROGTOBERFEST_IGNORE_LABEL_LIST = ['invalid'];
+
+// Makes The Array case insensitive for checking
+const labelIgnoreRegex = new RegExp(FROGTOBERFEST_IGNORE_LABEL_LIST.join('|'), 'i');
```

Comment:
Regex to check for invalid label is probably not a good idea. A simple string comparison should be enough.